app/components/organisation/addOrganisationSchema.ts

```diff
@@ -0,0 +1,67 @@
+import {JSONSchema6} from 'json-schema';
+
+const addOperatorSchema: JSONSchema6 = {
+  type: 'object',
+  title: 'Operator Details',
+  properties: {
+    operatorName: {
```

Comment:
Similarly to the facility, I would only keep the operator name in here (and maybe the CRA number)